src/egon/data/datasets/power_plants/pv_rooftop.py

```diff
@@ -129,6 +155,18 @@ def pv_rooftop_per_mv_grid_and_scenario(scenario, level):
             """
         ).capacity[0]
 
+        dataset = config.settings()["egon-data"]["--dataset-boundary"]
+
+        if dataset == "Schleswig-Holstein":
+            # since the required data is missing for a SH run, it is implemented
+            # manually here
+            total_2035 = 84070
+            sh_2035 = 2700
```

Comment:
Can't this value be obtained from scenario data? How fragile is this, e.g. does it still work for the new MaStR data?